crates/database/src/solver_competition_v2.rs

```diff
@@ -71,17 +71,18 @@ pub async fn load_by_tx_hash(
     tx_hash: TransactionHash,
 ) -> Result<Option<SolverCompetition>, sqlx::Error> {
     const FETCH_AUCTION_ID: &str = r#"
-        SELECT auction_id
+        SELECT s.auction_id
         FROM settlements s
         LEFT OUTER JOIN settlement_observations so ON
```

Comment:
The same can be achieved with a regular `JOIN`, right? I copied that part over from the old endpoint without much scrutiny.
```suggestion
        JOIN settlement_observations so ON
```